client/me/profile-links-add-wordpress/site.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Site from 'blocks/site';
+import { recordCheckboxEvent } from 'me/event-recorder';
+
+class ProfileLinksAddWordPressSite extends Component {
+	static propTypes = {
+		onChange: PropTypes.func.isRequired,
+		onSelect: PropTypes.func.isRequired,
+		site: PropTypes.object.isRequired,
+		checked: PropTypes.bool,
+	};
+
+	static defaultProps = {
+		checked: false,
+	};
+
+	inputName = `site-${ this.props.site.ID }`;
```

Comment:
That shouldn't be a problem, because we're mounting one `ProfileLinksAddWordPressSite` instance for each site anyway. But it's a nice improvement, so I've addressed in https://github.com/Automattic/wp-calypso/pull/14758/commits/f65e2af8f7c0b20c3b47f97fef522ed7a7abee26.